src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -580,6 +581,14 @@ public function testDumpDateTime($dateTime, $expected)
         $this->assertSame($expected, Inline::dump($dateTime));
     }
 
+    /**
+     * @requires PHP 8.1
+     */
+    public function testDumpUnitEnum()
+    {
+        $this->assertSame('Symfony\Component\Yaml\Tests\Fixtures\FooUnitEnum::BAR', Inline::dump(FooUnitEnum::BAR));
```

Comment:
shouldn't this be `!php/const etc`?

Reply:
You're right. Fixed the test and `dump` method as well.